### ISSUE TITLE
Application Error Message when resizing window (ubuntu 22.04)

### ISSUE THREAD

**Issue by otetLopez** [dracula/gitk#11]
![image](https://user-images.githubusercontent.com/23637735/207955598-d80807dd-15a8-4325-90a9-cf8a14be8abf.png)

**Comment by aaryamanb** [comment]
bump

**Comment by Alex-Beh** [comment]
> I've narrowed the offending config line to:
> `set want_ttk 0`
> By removing it, gitk automatically fills it up (after a launch) with:
> `set want_ttk 1`

Reference: https://github.com/dracula/gitk/issues/10

**Comment by lourot** [comment]
Thank you all! Out of curiosity, then I suppose with `set want_ttk 1` some parts of the UI aren't following the theme, as described in https://github.com/dracula/gitk/issues/4#issuecomment-553790057 ?

**Comment by miroslav-suvada** [comment]
Same issue. I'm running gitk v2.34.1. So after copying gitk to ~/.config/git/ and running gitk I've got same error as mentioned above and config got updatated like so
```diff
@@ -1,3 +1,21 @@
+set mainfont {sans 9}
+set textfont {monospace 9}
+set uifont {sans 9 bold}
+set tabstop 8
+set findmergefiles 0
+set maxgraphpct 50
+set maxwidth 16
+set cmitmode patch
+set wrapcomment none
+set autoselect 1
+set autosellen 40
+set showneartags 1
+set maxrefs 20
+set visiblerefs {"master"}
+set hideremotes 0
+set showlocalchanges 1
+set datetimeformat {%Y-%m-%d %H:%M:%S}
+set limitdiffs 1
 set uicolor #44475a
 set want_ttk 0
 set bgcolor #282a36
@@ -8,9 +26,12 @@
 set diffcolors {#ff5555 #50fa7b #bd93f9}
 set mergecolors {#ff5555 #bd93f9 #50fa7b #bd93f9 #ffb86c #8be9fd #ff79c6 #f1fa8c #8be9fd #ff79c6 #8be9fd #ffb86c #8be9fd #50fa7b #ffb86c #ff79c6}
 set markbgcolor #282a36
+set diffcontext 3
 set selectbgcolor #44475a
 set foundbgcolor #f1fa8c
 set currentsearchhitbgcolor #ffb86c
+set extdifftool meld
+set perfile_attrs 0
 set headbgcolor #50fa7b
 set headfgcolor black
 set headoutlinecolor #f8f8f2
@@ -31,3 +52,13 @@
 set linkfgcolor #bd93f9
 set circleoutlinecolor #44475a
 set diffbgcolors {{#342a36} #283636}
+set web_browser xdg-open
+set geometry(main) 1536x978+35+64
+set geometry(state) normal
+set geometry(topwidth) 1534
+set geometry(topheight) 225
+set geometry(pwsash0) "484 1"
+set geometry(pwsash1) "727 1"
+set geometry(botwidth) 600
+set geometry(botheight) 741
+set permviews {}
```
When I change the line to `set want_ttk 1` I've got no more error messages when resizing the window and it looks like this
![image](https://github.com/dracula/gitk/assets/23190737/b35e9bb8-5db8-4761-a3c7-50ff3e8bab1a)

**Comment by aaryamanb** [comment]
Same issue as @miroslav-suvada when trying: "set want_ttk 1" fix. 

The middle part of the window and outlines revert to light mode. Currently I've been dealing with the issue by skipping all errors and just not resizing any windows ...

**Comment by lourot** [comment]
The issues preventing us from using `set want_ttk 0` have been fixed in
* https://github.com/git/git/pull/1219
* https://github.com/git/git/commit/ef9b086d

The first version containing this fix is `v2.37.0`. On Ubuntu for example it's
available starting from Ubuntu 22.10 (Kinetic):
https://packages.ubuntu.com/kinetic/gitk

I have just tried it out and everything works well now for me.
